### ISSUE TITLE
aur package installation failing - sha256 checksums

### ISSUE THREAD

**Issue by O2-AC** [cloudlena/pacmanfile#2]
I just want to inform you, that the current version 0.0.7 is not installing through the aur, due to a checksum failure. You probably have to update the sha256 checksum in the PKGBUILD.

**Comment by cloudlena** [comment]
Hmm weird... For me, it's installing. Which distro are you on (is it vanilla arch?) and which AUR helper are you using?

**Comment by O2-AC** [comment]
Interesting, I use vanilla arch with paru as AUR helper and get the following error thrown.
```
1 aur/pacmanfile 0.0.7-1 [+0 ~0.00] [Installed: 0.0.6-2]
    Manage your pacman packages declaratively
:: Packages to install (eg: 1 2 3, 1-3):
:: 1
:: Resolving dependencies...
:: Calculating conflicts...
:: Calculating inner conflicts...

Aur (1) pacmanfile-0.0.7-1

:: Proceed to review? [Y/n]:

:: Downloading PKGBUILDs...
 PKGBUILDs up to date
 nothing new to review
fetching devel info...
==> Making package: pacmanfile 0.0.7-1 (Mon 07 Jun 2021 04:58:58 PM CEST)
==> Retrieving sources...
  -> Found pacmanfile
==> Validating source files with sha256sums...
    pacmanfile ... FAILED
==> ERROR: One or more files did not pass the validity check!
error: failed to download sources for 'pacmanfile-0.0.7-1':
:: Packages failed to build: pacmanfile-0.0.7-1
```

More fascinating is the fact, that if I manually download pacmanfile from:
`https://raw.githubusercontent.com/mastertinner/pacmanfile/v0.0.7/pacmanfile` and then run:
```
$ sha256sum pacmanfile
6179e2bc7312323edd60244cf4c1fa87b7850a9f0500bca0c956064b1d97590f  pacmanfile
```
I get the checksum provided in the PKGBUILD.

Edit: There also seems to be another user with that problem who commented on the AUR.

**Comment by O2-AC** [comment]
Update:
I got the install working by removing the old version from the paru cache (`~/.config/paru/clone/pacmanfile`). I assume paru used the PKGBUILD from the old version and thus was not able to verify the checksum.
I have no clue if that is a problem with paru or with the way the package was updated.

**Comment by cloudlena** [comment]
Maybe that was because I didn't actually release versions on GitHub to download from until 0.0.7. Therefore, I hope that this problem is resolved with the next release. If not, I'll have to investigate further :)

**Comment by cloudlena** [comment]
I did actually investigate further and it should be fixed with `0.0.7-3` now :)
